.env

```diff
@@ -0,0 +1,3 @@
+POSTGRES_USER=
```

Comment:
these should be the supported variables

POSTGRESQL_USER
POSTGRESQL_PASSWORD
POSTGRESQL_DATABASE
POSTGRESQL_ADMIN_PASSWORD